tasks/vmpooler.rb

```diff
@@ -5,8 +5,11 @@
 require 'puppet_litmus'
 require_relative '../lib/task_helper'
 
-def provision(platform, inventory_location)
+def provision(platform, inventory_location, vars)
   include PuppetLitmus::InventoryManipulation
+  unless vars.nil?
+    vars_hash = YAML.safe_load(vars)
```

Comment:
Is there a possibility for this to cause an unhandled exception ?

Reply:
Yes there is, good question @carabasdaniel ! However what are we going to do with it ? There isnt much to do, so just letting it fall out the bottom of the script is probably enough. The task will error if any exceptions are thrown.